diktat-common/src/main/kotlin/org/cqfn/diktat/common/config/rules/RulesConfigReader.kt

```diff
@@ -116,6 +116,13 @@ data class CommonConfiguration(private val configuration: Map<String, String>?)
         configuration?.get("domainName")
     }
 
+    /**
+     * Get version of kotlin from configuration
+     */
+    val kotlinVersion: String by lazy {
+        configuration?.get("kotlinVersion") ?: KotlinVersion.CURRENT.toString()
```

Comment:
Let's make it nullable or have some special value for missing version. If user didn't specify version, it'll be better if we do not run rules that require higher version, while having `KotlinVersion.CURRENT` as default will implicitly enable all rules

Reply:
Fixed, but if the user hasn't specified an option, why should we disable the rule? What if he doesn't know that such a feature is available with the new version?

Reply:
That's a good point. Let's log a warning if it's not present. Otherwise we'll, for example, add trailing commas everywhere in the code, while user is still using kotlin 1.3, and they'll need to rollback all these changes.